server/src/main/java/com/arcbees/beeshop/server/guice/DispatchServletModule.java

```diff
@@ -20,5 +17,7 @@ protected void configureServlets() {
         for (String locale : LocaleExtractor.SUPPORTED_LOCALES) {
             serveRegex("/" + locale, "/" + locale + "/").with(RootServlet.class);
         }
+
+        serve("/*").with(NotFoundServlet.class);
```

Comment:
I probably don't understand the context of your pull request well enough, but why didn't you do
`bindConstant().annotatedWith(DefaultPlace.class).to(NameTokens.NOT_FOUND);` in your Gin Module
instead?